src/clj/rems/form_validation.clj

```diff
@@ -18,14 +18,22 @@
        (not (str/blank? (:field/value field)))
        (not (re-matches util/+email-regex+ (:field/value field)))))
 
+(defn- invalid-option-value? [field]
+  (and (= (:field/type field) :option)
```

Comment:
Hmm this logic is correct but a bit hard to follow. Suggestion: always allow an empty value: we have separate logic for required/optional